src/cc2olx/olx.py

```diff
@@ -155,17 +155,38 @@ def process_ims_cc_filebase(item, html):
 
         return html
 
+    def _process_static_links_from_details(self, details):
+        """
+        Take a variable and recursively find & escape all static links within strings
+
+        Input: {'problem_details': '<img src="$IMS-CC-FILEBASE$/..."/> Lorem Ipsum', ...}
+        Output: {'problem_details': '<img src="/static/..."/> Lorem Ipsum', ...}
```

Comment:
There is a specific format in which you should write the docstring. Have a look at https://github.com/edx/cc2olx/blob/master/src/cc2olx/filesystem.py#L19